dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1269,6 +1274,91 @@ protected long indexCount(String query) {
         return searchRequestBuilder.execute().actionGet().getHits().getTotalHits();
 	}
 
+    @Override
+    protected long indexCount(final String query,
+                        final long timeoutMillis) {
+
+        final String queryStringQuery =
+                findAndReplaceQueryDates(translateQuery(query, null).getQuery());
+
+        // we check the query to figure out wich indexes to hit
+        IndiciesInfo info;
+
+        try {
+
+            info = this.indiciesAPI.loadIndicies();
+        } catch(DotDataException ee) {
+            Logger.fatal(this, "Can't get indicies information",ee);
+            return 0;
+        }
+
+        final Client client = new ESClient().getClient();
```

Comment:
You already have an instance variable `this.client` do you really want to instantiate again?

Reply:
ok